src/sentry/integrations/github/integration.py

```diff
@@ -477,48 +474,13 @@ class GitHubOpenPRCommentWorkflow(OpenPRCommentWorkflow):
 
     def safe_for_comment(self, repo: Repository, pr: PullRequest) -> list[dict[str, Any]]:
         client = self.integration.get_client()
-        logger.info(
-            _open_pr_comment_log(
-                integration_name=self.integration.integration_name, suffix="check_safe_for_comment"
-            )
-        )
         try:
             pr_files = client.get_pullrequest_files(repo=repo.name, pull_number=pr.key)
         except ApiError as e:
-            logger.info(
-                _open_pr_comment_log(
-                    integration_name=self.integration.integration_name, suffix="api_error"
-                )
-            )
-            if e.json and RATE_LIMITED_MESSAGE in e.json.get("message", ""):
-                metrics.incr(
-                    OPEN_PR_METRICS_BASE.format(
-                        integration=self.integration.integration_name, key="api_error"
-                    ),
-                    tags={"type": GithubAPIErrorType.RATE_LIMITED.value, "code": e.code},
-                )
-            elif e.code == 404:
-                metrics.incr(
-                    OPEN_PR_METRICS_BASE.format(
-                        integration=self.integration.integration_name, key="api_error"
-                    ),
-                    tags={"type": GithubAPIErrorType.MISSING_PULL_REQUEST.value, "code": e.code},
-                )
+            if (e.json and RATE_LIMITED_MESSAGE in e.json.get("message", "")) or e.code == 404:
+                return []
             else:
-                metrics.incr(
-                    OPEN_PR_METRICS_BASE.format(
-                        integration=self.integration.integration_name, key="api_error"
-                    ),
-                    tags={"type": GithubAPIErrorType.UNKNOWN.value, "code": e.code},
-                )
-                logger.exception(
-                    _open_pr_comment_log(
-                        integration_name=self.integration.integration_name,
-                        suffix="unknown_api_error",
-                    ),
-                    extra={"error": str(e)},
-                )
-            return []
+                raise
```

Comment:
Since we're failing loudly now, we're probably going to start retrying this task a lot more, are there any known exceptions we should ignore in the retry policy?

Reply:
There's no retry policy at the moment, which I think is okay